Haskell Day 2021, GitHub Organization, Slack Workspaceなどに適用されるGRC

igrep opened the pull request:
Fix https://github.com/haskell-jp/community/issues/44

https://github.com/haskell-jp/community/pull/29 でいただいた意見やAdminsでの議論に基づき、Haskell FoundationのGRC (Guidelines for Respectful Communication)をそのまま和訳して採用しようと思います。
（@nobsun, @kakkun61, 翻訳へのご協力ありがとうございました！）

ただし、翻訳したものをそのまま**Haskell-jpとして**使用すると実態と食い違う箇所ができてしまうので、 https://github.com/haskell-jp/community/commit/cb3ec882e78268190e4587c3f18895afae9b4188 で私が別途修正を加えました。

翻訳の内容などについて改めてみなさんから意見を集めたあと、修正してマージ次第、今年開催予定のHaskell Day 2021や、Haskell-jpのGitHub Organization・Slack Workspaceにおいて運用します。
（追記: すみません、前回の指摘事項が漏れてました。具体的には、以下のコミュニティーや、そこで開催されるイベントで運用します）

- https://github.com/haskell-jp/
- https://github.com/haskell-jp-admins/
- https://app.slack.com/client/T4LFB6C4C/ (haskell-jp.slack.com)
- https://techplay.jp/community/haskell-jp
- https://haskell-jp.connpass.com/

参考: Haskell FoundationにおけるGRCの原文: https://haskell.foundation/guidelines-for-respectful-communication/

igrep committed:
翻訳したGRCをそのまま追加

翻訳者: @nobsun, @kakkun61, @igrep

igrep committed:
Haskell-jp向けに修正

igrep commented:
念の為、 https://github.com/haskell-jp/community/pull/29#issuecomment-715336665 と同様のメンバーに加え、Adminsのまだメンションしていないメンバーをメンションします。

@takenobu-hs @minoki @matsubara0507 @mizunashi-mana @waddlaw @y-taka-23 @kayhide @cdepillabout @gksato @aruneko @khibino @eliza0x @haru2036 @as-capabl @maoe @YoshikuniJujo @Tatsuki-I @konn @egisatoshi @aiya000 @syocy @lotz84 @nakaji-dayo @fumieval

matsubara0507 commented on GRC.md:
```diff
@@ -0,0 +1,39 @@
+
+わたしたちの誰かがこの基準に反した場合の理想的な行動は、その人に個人的にメッセージを書き、その人の過ちにそっと目を向けさせることです。それに抵抗がある場合には、[日本Haskellユーザーグループ管理委員会（Haskell-jp Admins）](https://haskell.jp/blog/posts/about_admins.html#%E4%BB%8A%E5%BE%8C%E3%81%AE%E6%B4%BB%E5%8B%95%E3%81%A8%E9%80%A3%E7%B5%A1%E5%85%88)に連絡してください。
+
+通常、わたしたちの対応は謝罪してあなたが不幸に思っている行為をとめることです。わたしたちが誤解されていたり、不当に非難されていると感じたとしても、もっとうまく伝えられたことがある可能性が高いし、謝罪は反論よりも和解をもたらす可能性がはるかに高いのです。
```
ここだけよくわからず、何度も読み返したり原文読んだりしました。
とくに「わたしたちが誤解されたり～」の辺りが分かりにくかったのですが、これは「もし、（対応を求められた）わたしたちが誤解されていると感じたり、不当に避難されていると感じたりしたとしても、もっとうまく伝えられることがあった可能性は高いですし、謝罪は反論よりも和解をもたらす可能性がはるかに高い（ので謝罪するの）のです。」ですよね？

igrep replied:
だと思います。
私自身ちょっとここの理解には苦しみました 😰 
ここに残っていない私のコミットログでは

> 「わたしたちが誤解されていたり、不当に非難されていると感じたとしても、それはもっとうまく伝えられたことが**あるため**だろうし、謝罪は反論よりも和解をもたらす可能性がはるかに高いのです。」
> と、「誤解」や「不当な非難」が発生することについてなだめているような気がしますが、ちょっと意訳しすぎなきもしたのでここまでで。

と、いうことであまり意訳するのも好ましくないと思ってそのままにしてました。

fumieval replied:
私なりに訳してみました。

> Our response should usually be to apologise and stop doing what it was that you are unhappy about.

> 基本的に、不快にさせるようなことをやめ謝罪するという対応を取るべきです。

> Even if we feel we have been misinterpreted or unfairly accused, 

> たとえ誤解、あるいは不当な非難と感じたとしても、

> the chances are good there was something we could have communicated better, and an apology is far more likely to bring healing than is a counter-accusation.

> おそらく、よりうまく伝えられる余地があったでしょうし、言い返すよりは謝罪したほうが和解しやすいでしょう。

igrep replied:
ありがとうございます！そのまま採用したいので、直接このブランチにコミットしちゃってください！

igrep replied:
すみません、マージを急ぎたいので https://github.com/haskell-jp/community/pull/49/commits/8313b164c9e487e384cea6f16be550c51642f03a で私が適用しました。

fumieval replied:
すみません、見落としてました。ありがとうございます

mizunashi-mana commented on GRC.md:
```diff
@@ -0,0 +1,39 @@
+
+## 相互を尊重したコミュニケーションのためのガイドライン
+
+Haskellコミッティのメンバーとして、わたしたちは、公共の場で高い水準の行動をすることを約束します。わたしたちには、すべてをカバーする1つの原則があります。
```
nit: Haskell コミッティ -> Haskell コミュニティ？

kakkun61 replied:
元が Haskell Foundation なので「Haskell committee」でした。

我々はそうではないので削除がよさそう。

https://www.haskell.org/haskell-org-committee/

igrep replied:
「Haskellコミッティのメンバーとして、」を削除する、ということですか？それなら賛成 👍

kakkun61 replied:
> 「Haskellコミッティのメンバーとして、」を削除する、ということですか？

ういうい

igrep replied:
8d4d99f で対応。

igrep committed:
日本語版はHaskell Commiteeとしての約束ではない

Thanks mizunashi-mana, kakkun61!

igrep committed:
分かりづらい箇所をもっと自然に

Thanks, @fumieval

https://github.com/haskell-jp/community/pull/49#discussion_r631581382

igrep commented on GRC.md:
```diff
@@ -0,0 +1,39 @@
+- コミュニケーションにおいては、相手の情熱、専門知識、善意を一貫して尊重し肯定します。たとえ、時に、他者のこうした資質に疑いをもつことがあったとしても、無能、悪意あるいは下心を公に非難することはありません。
+- いつなんどきも礼儀を重んじるよう努めます。無礼、悪口、嫌がらせなどがあってはいけません。
+- 誰かと意見が合わない場合、対話の相手が、攻撃された、屈辱を受けた、軽蔑された、あるいは、疎外されたと感じるような表現は避けます。批判はつねに具体的な声明や主張に対するものであって、人に対するものであってはなりません。
+- 意見の相違そのものは構いません。堅牢な技術的議論がわれわれを豊かにしてくれます。しかし、その議論のトーンを、お互いを尊敬したり賞賛したりする人々の会話のトーンにすることを目指します。意見の相違があるところでは、相手の視点、目標、動機、優先順位などに興味を持つようにします。
```
なんか内容と訳が合わないな、と思ったら項目が分けられてなかった。

```suggestion
- 意見の相違そのものは構いません。堅牢な技術的議論がわれわれを豊かにしてくれます。しかし、その議論のトーンを、お互いを尊敬したり賞賛したりする人々の会話のトーンにすることを目指します。
- 意見の相違があるところでは、相手の視点、目標、動機、優先順位などに興味を持つようにします。
```

igrep committed:
項目が正しく分割されてないので修正
